### PR TITLE
FLUT-927176 - [Others] Added trademark symbol for installation and readme

### PR DESCRIPTION
**Pull request by AswiniDileep**
## Description ##

Updated trademark symbol to the Syncfusion's, Syncfusion and Syncfusion Essential Studio keywords in installation and  readme files.

**Readme**
|Before|After|
|--|--|
|image|image|
   
**Installation-errors**
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|
|image|image|
|image|image|

**how-to-download (mac-installer)**
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|    
|image|image| 
|image|image|

**how-to-install(mac-installer)**
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|
|image|image|
|image|image|

**how-to-download (linux-installer)**
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|   

**how-to-install(linux-installer)**
|Before|After|
|--|--|
|image|image|
|image|image|

**how-to-download (offline-installer)**
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|

**how-to-install(offline-installer)**
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|
|image|image|
|image|image|      
|image|image|
|image|image|

how-to-download (web-installer)
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|
|image|image|

**how-to-install(web-installer)**
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|
|image|image|
|image|image|
|image|image|
|image|image|   
|image|image|
|image|image|  
|image|image|
|image|image|

### CODE REVIEW

File: Flutter/installation/Installation-errors.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: post
 title: Installation Errors in Flutter - Syncfusion
 description: Learn here about the common installation errors and solution to those errors in Syncfusion Flutter Components installation.
-platform: Flutter
+platform: flutter
```
Comment (sfHariHaraSudhan) [inline Flutter/installation/Installation-errors.md:5]: Hi @AswiniDileep,

Why the casing of the Flutter is changed here?

Regards,
Hari Hara Sudhan. K.

File: Flutter/installation/Installation-errors.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: post
 title: Installation Errors in Flutter - Syncfusion
 description: Learn here about the common installation errors and solution to those errors in Syncfusion Flutter Components installation.
```
Comment (sfHariHaraSudhan) [inline Flutter/installation/Installation-errors.md:4]: Hi @AswiniDileep,

Check and ensure once with @LavanyaGowtham2021 Mam that we dont need to update trademark symbol here.

Regards,
Hari Hara Sudhan. K.

File: Flutter/installation/linux-installer/how-to-download.md [hunk 3]
```diff
@@ -28,12 +28,12 @@ Our 30-day trial can be downloaded in two ways.
 ### Download Free Trial Setup
 
 1. You can evaluate our 30-day free trial by visiting the [Download Free Trial](https://www.syncfusion.com/downloads) page and select the product
-2. After completing the required form or logging in with your registered Syncfusion account, you can download the trial installer from the confirmation page. (as shown in below screenshot.)
+2. After completing the required form or logging in with your registered Syncfusion<sup>&reg;</sup> account, you can download the trial installer from the confirmation page. (as shown in below screenshot.)
 
    ![Trial and downloads of Syncfusion Flutter](images/trial-confirmation.png)
```
Comment (sfHariHaraSudhan) [inline Flutter/installation/linux-installer/how-to-download.md:33]: Hi @AswiniDileep,

Trademark is not added in this hyperlink. Check and ensure it once.

Regards,
Hari Hara Sudhan. K.

**Review by sfHariHaraSudhan — APPROVED** [review]
Approved

File: Flutter/installation/linux-installer/how-to-download.md (hunk 3 above)
Comment (AswiniDileep) [reply Flutter/installation/linux-installer/how-to-download.md:33]: Hi @sfHariHaraSudhan,

As it is an image file not added as this text changes won't show as other text in preview else in web after changes also.
It is just hyperlink for image. 

Regards,
Aswini S.

File: Flutter/installation/Installation-errors.md (hunk 2 above)
Comment (AswiniDileep) [reply Flutter/installation/Installation-errors.md:4]: Okay.

**Pull request merged**